oximeter/db/src/client.rs

```diff
@@ -610,7 +610,7 @@ impl DbWrite for Client {
     /// Wipe the ClickHouse database entirely from a replicated set up.
     async fn wipe_replicated_db(&self) -> Result<(), Error> {
```

Comment:
That sounds like a good idea! I'll link this comment to the relevant issue

Reply:
So one additional note to add here, from a conversation I had with @davepacheco. It's probably a good idea in the long run to have this front-end runner have very little configuration itself, and instead get that all from Nexus. It would only need to know how to reach Nexus (via DNS) and probably an ID of some kind, and then hit an endpoint in Nexus with that ID. That request could return the configuration that node needs to run (addresses, ports, whether to start keepers and or data servers, etc.).